src/test/java/TestSyncSender.java

```diff
@@ -0,0 +1,17 @@
+package io.airbrake.log4javabrake2;
```

Comment:
Does this file really test anything?

Reply:
Good point, that's a bit misleading. This file has no tests but provides a kind of mock SyncSender. I'll experiment with renaming it something like `MockSyncSender` instead. Thanks for the review!